### DISCUSSION TITLE
Update to 2.5.282 from Dashboard crashes navigation

### DISCUSSION THREAD

**Discussion by Max2983** [requarks/wiki#5308] [Error / Bug Report] [answered]
I have WikiJS installed from Docker image at Digital ocean. Previous updates run smooth with no bugs. But today I have updated to the latest 2.5.282 and it crashed my navigation. I am using Classic navigation type ('file tree'). I have two "folders", and they are not clickable anymore. I can't get into any of them. How can I fix this? I am not a software engineer.

**Comment by NGPixel** [comment, marked as answer]
Duplicate of #5295
